schema/manifestlist_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
I haven't looked into #444 again, but I think `manifestlist_test.go` needs a success test with OCI-defined `mediaType`s for both the root `mediaType` and the `manifests[0].mediaType`.